.github/workflows/build.yaml

```diff
@@ -542,19 +545,23 @@ jobs:
         run: |
           make -C operator/ bundle bundle-build
 
-      - name: Build Operator image
+      - name: Setup operator build
+        # Skip for external contributions.
+        if: |
+          github.event_name == 'push' || !github.event.pull_request.head.repo.fork
         run: |
-          scripts/lib.sh retry 6 true make -C operator/ build docker-build
+          echo "DOCKER_BUILDX_ARGS=--push" >> "$GITHUB_ENV"
 
-      - name: Check that Operator image is runnable
-        run: docker run --rm "quay.io/rhacs-eng/stackrox-operator:$(make --quiet --no-print-directory -C operator tag)" --help
```

Comment:
Removing this. It will be checked by e2e tests.
The problem is that docker buildx cannot push multiarch images to the local docker engine (with `--load`)
So if the images are not pushed (e.g if CI is triggered by a fork), then we can't pull them locally to see if they're runnable